dvc/exceptions.py

```diff
@@ -261,6 +261,17 @@ def __init__(self, target_infos, stats=None):
                 "\n".join(targets), error_link("missing-files"),
             )
         )
+
+        if failed_not_stored:
+            m = (
+                "The following outputs are marked as `store: false` so "
+                "remote storage was disabled, and are not present in the "
+                "cache:\n{}\n{}\n".format(
```

Comment:
```suggestion
                "The following outputs are marked as `store: false` (which"
                "disables remote storage) and they are not present in the "
                "cache:\n{}\n{}\n".format(
```

Reply:
> The following outputs are marked as `store: false` so ...
foo
\<https://error.dvc.org/checkout-failed-no-store\>
Checkout failed for following targets:
foo
Is your cache up to date?
\<https://error.dvc.org/missing-files\>

Hmmm yeah this is a little confusing. I guess it's best to have too much info rather than too little but ideally this should be closer to

> Checkout failed for **the** following targets:
> \* foo (marked with `store: false` in foo.dvc)
> \* bar
>
> Is your cache up to date?
\<https://error.dvc.org/missing-files\>
>
> Note that `store: false` outputs are not searched for in remote storage, only in the cache, see \<https://error.dvc.org/checkout-failed-no-store\>.

But even that is a little long... Not sure! I'm not trying to block this merge either, could be addressed separately I suppose.

Reply:
Done! It's now

```
Checkout failed for following targets:
foo (marked with 'store: false' in dvc file)

Is your cache up to date?
<https://error.dvc.org/missing-files>

Note that 'store: false' outputs are not searched for in remote storage, only in the cache. See <https://error.dvc.org/checkout-failed-no-store>
```

I didn't include a link to the DVC file since that'd involve a bit of refactoring which didn't seem worth it.

Reply:
Lovely